broadinstitute/single_cell_portal_core
# Upload wizard expression files (including bundles)  (SCP-3657)

SCP-3657, SCP-3656.  This adds expression file upload to the new upload wizard, and also supports bundling.  the UX got a 'seems ok' nod from Jerome, so it might change a bit later, but nothing major.  

![image](https://user-images.githubusercontent.com/2800795/135153205-a10fe3d9-3737-4cd7-a474-e4784d28a3ed.png)

MANUAL TESTING:
1. Create a new study (make sure react_upload_wizard feature flag is on)
2. confirm you are taken to the raw counts upload
3. upload one or more raw counts files (including an mtx+barcodes+genes file)
4. upload one or more expression files (including an mtx+barcodes+genes file)
5. confirm parse jobs are launched and ingest succeeds/fails appropriately
